UI file created

jeevankishorem opened the pull request:
Dear Matt,
Check out this commit with UI file.

jeevankishorem committed:
Merge pull request #1 from mwcarlis/master

Update from original

jeevankishorem committed:
I have added the UI with the templates to call functions for active passive and independent reconnaissance activity

jeevankishorem committed:
Revert "I have added the UI with the templates to call functions for active passive and independent reconnaissance activity"

This reverts commit a87955058b7c7a6da5eb56941ffab25bc7ebad47.

jeevankishorem committed:
Revert "Revert "I have added the UI with the templates to call functions for active passive and independent reconnaissance activity""

This reverts commit 1b969566e626e6ca9c05975cd4eedffdcc8affde.

jeevankishorem committed:
commit with one UI file

mwcarlis commented:
Hi.

This is looking good Jeevan.

I will start to merge it in soon, but I need to finish writing a paper
first and do an assignment for my compiler class.

-Matt

On Mon, Nov 30, 2015 at 1:51 PM, jeevankishorem <notifications@github.com>
wrote:

> Dear Matt,
> Check out this commit with UI file.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/mwcarlis/reconnaissance_engine/pull/1
> Commit Summary
>
>    - Merge pull request #1 from mwcarlis/master
>    - I have added the UI with the templates to call functions for active
>    passive and independent reconnaissance activity
>    - Revert "I have added the UI with the templates to call functions for
>    active passive and independent reconnaissance activity"
>    - Revert "Revert "I have added the UI with the templates to call
>    functions for active passive and independent reconnaissance activity""
>    - commit with one UI file
>
> File Changes
>
>    - *A* UI.py
>    <https://github.com/mwcarlis/reconnaissance_engine/pull/1/files#diff-0>
>    (65)
>
> Patch Links:
>
>    - https://github.com/mwcarlis/reconnaissance_engine/pull/1.patch
>    - https://github.com/mwcarlis/reconnaissance_engine/pull/1.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mwcarlis/reconnaissance_engine/pull/1>.
>

jeevankishorem commented:
Hey Matt. Sure take your time. I will be able to contribute more from Thursday.  

Sent from my iPhone

> On Nov 30, 2015, at 2:02 PM, mwcarlis <notifications@github.com> wrote:
> 
> Hi.
> 
> This is looking good Jeevan.
> 
> I will start to merge it in soon, but I need to finish writing a paper
> first and do an assignment for my compiler class.
> 
> -Matt
> 
> On Mon, Nov 30, 2015 at 1:51 PM, jeevankishorem <notifications@github.com>
> wrote:
> 
> > Dear Matt,
> > Check out this commit with UI file.
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/mwcarlis/reconnaissance_engine/pull/1
> > Commit Summary
> >
> > - Merge pull request #1 from mwcarlis/master
> > - I have added the UI with the templates to call functions for active
> > passive and independent reconnaissance activity
> > - Revert "I have added the UI with the templates to call functions for
> > active passive and independent reconnaissance activity"
> > - Revert "Revert "I have added the UI with the templates to call
> > functions for active passive and independent reconnaissance activity""
> > - commit with one UI file
> >
> > File Changes
> >
> > - *A* UI.py
> > <https://github.com/mwcarlis/reconnaissance_engine/pull/1/files#diff-0>
> > (65)
> >
> > Patch Links:
> >
> > - https://github.com/mwcarlis/reconnaissance_engine/pull/1.patch
> > - https://github.com/mwcarlis/reconnaissance_engine/pull/1.diff
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/mwcarlis/reconnaissance_engine/pull/1>.
> >
> —
> Reply to this email directly or view it on GitHub.
>
